pkg/machines/scripts/create_machine.sh

```diff
@@ -85,13 +90,21 @@ else
     fi
 fi
 
+
+if [ "$CHOOSE_STORAGE" = "true" ]; then
```

Comment:
Isn't it by default on when you pass the volume?
From man virt-install: Most checks are performed by default... Please verify that this check is not redundant.